crawl.sh

```diff
@@ -8,7 +8,7 @@ time wget \
     --reject '*.css,*.doc,*.docx,*.gif,*.ico,*.jpg,*.js,*.mp3,*.pdf,*.png,*.txt,*.wav,*.woff,*.woff2,*.xls,*xlsx,*.zip' \
     --reject-regex "topics=|authors=|categories=|filter_blog_category=|ext_url=|search_field=|issuer_name=" \
     --recursive \
-    --level=inf \
+    --level=140 \
```

Comment:
I'm hoping to slowly step up the crawl level in a separate PR. We've never had success with anything other than level 3, so I think I'll try level 4 or 5 today.